src/redux/book/booksSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
```

Comment:
- [ ] According to the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/use_redux_in_react_components_v2.md) please ensure that an initialState variable for your booksSlice, includes the books objects. In addition, please display your books, received from the slice, in a reusable component ☺️ instead of displaying the books from data array.